### PR TITLE
fix: Set HubCard Card variant

### PR DESCRIPTION
**Pull request by codecaaron**
### Overview

* It won't ever be anything else.

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.73-alpha.904b4e.0<br/>@codecademy/gamut-labs@19.26.2-alpha.904b4e.0<br/>@codecademy/styleguide@44.3.2-alpha.904b4e.0<br/></pre>

File: packages/gamut-labs/src/HubCard/index.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ export const HubCard: React.FC<HubCardProps> = ({
   backgroundImage,
 }) => (
   <Card
+    variant="white"
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/HubCard/index.tsx:18]: I don't think this is correct? We've been talking about dark mode all over.

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://613bb86bf567d93e0987d49e--gamut-preview.netlify.app>https://613bb86bf567d93e0987d49e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/613bb86bf567d93e0987d49e>Deploy Logs</a></p>

File: packages/gamut-labs/src/HubCard/index.tsx (hunk 1 above)
Comment (codecaaron) [reply packages/gamut-labs/src/HubCard/index.tsx:18]: HubCards are only ever used with white backgrounds currently.  And do not have a dark mode analogue, same with curriculum cards in current designs (IE Progress has specific color states)

Definitely something we need design to take a look at /  figure out what we want to do.

Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/HubCard/index.tsx:18]: I filed a Hackathon item heh https://www.notion.so/codecademy/Dark-mode-in-the-catalog-Docs-983fb451cbfa491c8a04f5d24b768d76

**Review by dreamwasp — APPROVED** [review]
it wont ever be anything else

**Review by JoshuaKGoldberg — APPROVED** [review]
🍦

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.76-alpha.92453f.0<br/>@codecademy/gamut-labs@19.26.5-alpha.92453f.0<br/>@codecademy/styleguide@44.5.1-alpha.92453f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6141044b878ee20a5bf6f626--gamut-preview.netlify.app>https://6141044b878ee20a5bf6f626--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6141044b878ee20a5bf6f626>Deploy Logs</a></p>

**Pull request merged**